crates/matrix-sdk/src/client/mod.rs

```diff
@@ -913,6 +916,18 @@ impl Client {
             .collect()
     }
 
+    /// Get a stream of all the rooms, in addition to the existing rooms.
```

Comment:
same here + this should be fixed up into bc4e0f35a1fc72f37648bdeb36c9135088006e57?